### PR TITLE
BDD test for insights content service

### PR DESCRIPTION
**Pull request by dpensi**
# Description

BDD test for insights content service

Fixes CCXDEV-7542

## Type of change

Please delete options that are not relevant.

- New test scenario added into existing feature file

## Testing steps
checkout the branch and run
`make insights-content-service`

## Checklist
* [x] Pylint passes for Python sources
* [x] sources has been pre-processed by Black
* [x] updated documentation wherever necessary

### CODE REVIEW

File: features/steps/common_http.py [hunk 1]
```diff
@@ -187,3 +188,12 @@ def check_for_null_attribute(context, attribute):
 
     assert attribute in json, f"Attribute {attribute} is not returned by the service"
     assert json[attribute] is None, f"Attribute {attribute} should be null"
+
+
+@given(u'the service is running')
+@then(u'the service is running')
+def is_server_running(context):
```
Comment (epapbak) [inline features/steps/common_http.py:195]: I'm not a fan of this step definition. So far what we've been doing is starting the service on each scenario with the corresponding arguments. Even if content-.service is a simpler service, I'd do it the same way.

File: requirements/insights-content-service.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+behave
+pytest
+psycopg2
+jsonschema
+requests
+minio
```
Comment (epapbak) [inline requirements/insights-content-service.txt:6]: Please remove from this file the requirements that are not related to these tests (minio, psycopg2, etc.) and install both `requirements/requirements.txt` and `requirements/insights-content-service.txt` in the runner script.

Comment (dpensi) [reply requirements/insights-content-service.txt:6]: if I remove for example `minio` I get:

`Exception ModuleNotFoundError: No module named 'minio'
Traceback (most recent call last):

.......

File "/home/dpensier/python/github.com/dpensi/insights-behavioral-spec/features/src/minio.py", line 17, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
`

**Review by tisnik — APPROVED** [review]
LGTM
